dualscreeninfo/package.json

```diff
@@ -36,20 +36,19 @@
     "@babel/runtime": "*",
     "react": "*",
     "react-native": "*",
-    "react-native-windows":  "*"
+    "react-native-windows": "*"
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
     "@react-native-community/bob": "^0.9.7",
     "@types/jest": "^25.1.1",
-    "@types/react-native": "^0.61.10",
+    "@types/react": "^16.8.6",
+    "@types/react-native": "^0.61.17",
     "babel-jest": "^25.1.0",
     "jest": "^25.1.0",
     "metro-react-native-babel-preset": "^0.58.0",
     "prettier": "^1.19.1",
-    "react": "16.8.6",
-    "react-native": "^0.60.6",
     "react-native-windows": "^0.60.0-vnext.127",
```

Comment:
>"react-native-windows": "^0.60.0-vnext.127", [](start = 4, length = 44)

react-native-windows version 60 isn't compatible with react-native  version 61.  This should be updated to ^0.61.0-beta.28

Reply:
fixed